scripts/check_e2e_tests_are_captured_in_ci.py

```diff
@@ -23,12 +23,11 @@
 import re
 
 import python_utils
-import utils
 
-# These test suites are not present in travis CI. One is extra
+# These test suites are not present in Action. One is extra
 # (ie. (full: [*.js])) and other test suites are being run by CircleCI.
-TEST_SUITES_NOT_RUN_ON_TRAVIS = [
-    'full', 'accessibility', 'adminPage', 'classroomPage',
```

Comment:
Why is `classroomPage` not in here? It's in the CircleCI config

Reply:
We run classroomPage in actions as well.